CSV Content function not being imported in Typescript

Hello,
I tried importing 'react-csv-downloader/dist/lib/csv' and using it directly.

However, the path does not seem to be present. Getting the following error: 
"Cannot find module 'react-csv-downloader/dist/lib/csv' or its corresponding type declaration"

Edit:
I found the csv function in the following path
'react-csv-downloader/dist/cjs/lib/csv'

Please consider updating the doc

--- a/README.md
+++ b/README.md
@@ -156,7 +156,7 @@ render() {
 
 ## Get CSV contents
 
-If you just need to get CSV contents, import `react-csv-downloader/dist/lib/csv` function and use it directly.
+If you just need to get CSV contents, use `import { toCsv } from 'react-csv-downloader';` to import toCsv function and use it directly.
 
 ## License
 

--- a/src/index.tsx
+++ b/src/index.tsx
@@ -3,6 +3,8 @@ import * as React from 'react'
 
 import toCsv, { ICsvProps } from './lib/csv'
 
+export { toCsv, ICsvProps }
+
 export type PrefixSuffix = boolean | string | number
 
 export interface ICsvDownloadProps

